hawkbit-rest-core/pom.xml

```diff
@@ -48,7 +48,6 @@
       <dependency>
          <groupId>javax.servlet</groupId>
          <artifactId>javax.servlet-api</artifactId>
-         <scope>provided</scope>
```

Comment:
I removed it for symmetry reasons, as the hawkbit-security-core also uses compile scope for the javax.servlet-api.
Using provided scopes brings the risk of runtime problems regarding wrong class version (e.g. gwt-elemental brings transitive dependency on gwt-user that brings along classes of javax.servlet-api with wrong version; this is no current problem, but dependencies might change...).
On the other hand: the usage of compile scope might cause clashes in classpath.
I don't know what is better, but I think hawkbit-security-core and hawkbit-rest-core should handle it symmetrically.